drop-transfer/src/ws/server/mod.rs

```diff
@@ -905,15 +905,7 @@ impl FileXferTask {
 
                     match result {
                         Err(crate::Error::Canceled) => {
-                            info!(logger, "File {} cancelled", self.file.id());
-
-                            if let Err(err) = state
-                                .transfer_manager
-                                .incoming_download_cancel(self.xfer.id(), self.file.id())
-                                .await
-                            {
-                                warn!(logger, "Failed to store download finish: {err}");
-                            }
```

Comment:
Could you explain on how this was a bug?

Reply:
Ah i see, thank you for explaining

True about QA, perhaps they're still doing more surface tests